app/services/api/external-api/scenes/scene-item.ts

```diff
@@ -208,23 +213,25 @@ export class SceneItem extends SceneNode implements ISceneItemActions, ISceneIte
   }
 
   flipX(): void {
+    // return this.sceneItem.flipX(this.sceneItem.display);
     return this.sceneItem.flipX();
   }
 
   flipY(): void {
+    // return this.sceneItem.flipY(this.sceneItem.display);
```

Comment:
not sure why this and the above are commented, does this function not require the display?

Reply:
Removed. I realized later that I could get the display from the scene item internally for this.